pkg/etcdutil/etcdutil.go

```diff
@@ -129,6 +129,25 @@ func EtcdKVGet(c *clientv3.Client, key string, opts ...clientv3.OpOption) (*clie
 	return resp, nil
 }
 
+// EtcdKVDelete returns the etcd DeleteResponse by given key or key prefix
+func EtcdKVDelete(c *clientv3.Client, key string, opts ...clientv3.OpOption) (*clientv3.DeleteResponse, error) {
+	ctx, cancel := context.WithTimeout(c.Ctx(), DefaultRequestTimeout)
+	defer cancel()
```

Comment:
I think it's more appropriate to use a context passed in.

```suggestion
func EtcdKVDelete(ctx context.Context, c *clientv3.Client, key string, opts ...clientv3.OpOption) (*clientv3.DeleteResponse, error) {
```